Sources/libcggen/BCCGGenerator.swift

```diff
@@ -139,7 +139,7 @@ private func generateSteps(steps: [DrawStep], context: Context) -> [UInt8] {
     case let .moveTo(to):
       return byteCommand(.moveTo, to)
     case let .curveTo(c1, c2, end):
-      return byteCommand(.curveTo, c1, c2, end)
+      return byteCommand(.curveTo, end, c1, c2)
```

Comment:
It is good to keep order in bytecode and DrawStep on sync, so it’s better to change it in DrawStep, or on interpreter side.